README.md

```diff
@@ -14,12 +14,22 @@ Available variables are listed below, along with default values (see `defaults/m
 
     fluentd_package_state: present
 
+Extra groups to add log access, secret:
```

Comment:
What you mean by `, secret`?

Reply:
For exemple, to read datadog config file for vault, user must be in `vault-datadog` group

Reply:
Then I would rephrase like this:
```suggestion
Extra groups to give specific accesses to fluentd user, like logs or secret:
```